src/index.ts

```diff
@@ -57,7 +57,7 @@ export default <Module> function () {
 
     this.options.build!.transpile!.push(...componentDirs.filter(dir => dir.transpile).map(dir => dir.path))
 
-    let components = await scanComponents(componentDirs)
+    let components = await scanComponents(componentDirs, this.options.srcDir as string)
```

Comment:
`srcDir` type being `string | undefined`, using `!` (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html -> Non-null assertion operator) make it resolved to type `string`, which is cleaner :)

```suggestion
    let components = await scanComponents(componentDirs, this.options.srcDir!)
```

Reply:
Didn't know that thanks for the tip 😊 BTW why it is nullable? In nuxt context it always has value

Reply:
@pi0 It's cause `options` is of type `NuxtConfiguration`, I once tried to make it Required<NuxtConfiguration> to skip nullables but it applies only to top level properties.

Reply:
I will message you re this ;)